queries.sql

```diff
@@ -15,3 +15,44 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/* Project Day 2 */
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species is NULL;
+
+DELETE FROM animals;
+
+rollback;
+
+SELECT * FROM animals;
```

Comment:
- [ ] Kindly make sure you write here the exact command you run to make the task done. a transaction is supposed to start with a BEGIN.
```suggestion
BEGIN;
DELETE FROM animals;

rollback;

SELECT * FROM animals;
```